erc20-load-test-tool/src/main/java/com/vmware/ethereum/service/WorkloadCommand.java

```diff
@@ -76,29 +78,50 @@ public void run() {
     }
   }
 
-  /** Transfer token asynchronously. */
+  /** Transfer batched Transactions asynchronously. */
   public CompletableFuture<BatchResponse> transferBatchAsync(BatchRequest batchRequest) {
     Instant startTime = now();
     log.debug("batch requests added");
 
     return api.transferBatchAsync(batchRequest)
-        .whenComplete((response, throwable) -> receiptProcessor(startTime, response, throwable));
+        .whenComplete(
+            (response, throwable) -> {
+              if (throwable != null) {
+                if (throwable instanceof JsonRpcError) {
```

Comment:
Except this if/else, everything else is not related to this Jira (BC-18137). So I believe, this is refactoring, which is good but make sure you have tested it well.

Reply:
Yes, most of the changes are related to refactoring and exception handling. I have been testing it thoroughly.